scripts/variantstore/wdl/extract/strip_custom_annotations_from_sites_only_vcf.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import sys
+
+def strip_custom_annotations_from_sites_only_vcf(input_sites_only_vcf, input_custom_annotations_tsv, output_vcf, output_custom_annotations_tsv):
+    with open(input_custom_annotations_tsv) as tsv:
+        header, column_names, names_to_column_map = parse_annot_header(tsv, input_custom_annotations_tsv)
+        with open(output_custom_annotations_tsv, 'w') as out_tsv:
+            out_tsv.write(header)
+            with open(input_sites_only_vcf) as input_vcf:
+                vcf_header, chrom_line = skip_vcf_header(input_vcf)
+                with open(output_vcf, 'w') as out_vcf:
+                    out_vcf.write(vcf_header)
+
+                    if not chrom_line.endswith("INFO"):
+                        print(f"ERROR: Input Sites only VCF {input_sites_only_vcf} has unexpected header")
+                        sys.exit(1)
+
+                    while True:
+                        vcf_line = input_vcf.readline().rstrip()
+                        if (vcf_line == ""):
+                            print("Done")
+                            sys.exit(0)
+
+                        vcf_fields = vcf_line.split("\t")
+                        info_field_map = get_info_fields(vcf_fields[7])
+
+                        annotations = []
+                        annotations.append(vcf_fields[0])
+                        annotations.append(vcf_fields[1])
+                        annotations.append(vcf_fields[3])
```

Comment:
2 missing on purpose?

Reply:
Yes, field 2 is the 'ID' field in the VCF, not present in the annotations file (nor even populated in our sites only VCF)